libs/core/src/scss/components/_grid.scss

```diff
@@ -0,0 +1,97 @@
+@use 'sass:map';
+
+/* stylelint-disable custom-property-pattern */
+$breakpoint-map: (
+  'xs': 0,
+  'sm': 632px,
+  'md': 768px,
+  'lg': 992px,
+  'xl': 1200px,
+);
+
+.kirby-grid {
+  --kirby-grid-columns: 12;
+  --kirby-grid-columnSpacing: 24px;
+  --kirby-grid-rowSpacing: 24px;
+
+  min-width: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: row wrap;
+  margin: calc(var(--kirby-grid-rowSpacing) / -2) calc(var(--kirby-grid-columnSpacing) / -2);
+
+  &[spacing='1'],
+  &[data-spacing='1'] {
+    --kirby-grid-columnSpacing: 8px;
+    --kirby-grid-rowSpacing: 8px;
```

Comment:
UI/UX question: Do you know if it has been considered if we want the spacing to scale together with text, if users use bigger text sizes in their browser? 

Would be trivial to base spacing on rem units instead, if that is desired behavior. 
If we want the spacing to be consistent across any font-size, discard this comment.

Reply:
I would expect no as I the definitions of the breakpoint is made to respect desired min width for columns. I also think this would add complexity for the component as on small devices where we would need more space for the bigger text we would now also use more space for the gap :-) ?

Reply:
Good point, lets leave it at using our existing design tokens here 👍